src/linkedlist.js

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
+class Node {
+
+  constructor( data ) {
+    this.data = data
+    this.next = null
+  }
+
+}
+export default class LinkedList {
+
+  constructor () {
+      this.head = null
+      this.tail = null
+      this.count = 0
+
+
+  }
+  // Inserts a node (with the provided data) to the head of the list
+  insertFirst ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if head is defined
+    if ( this.head ) {
+      //set old head to next of new node
+      node.next = this.head
+      //if head is defined, set new node to head
+    } else {
+      //if not defined set current node to head
+      this.tail = node
+    }
+    this.head = node
+    this.count++
+  }
+  // Returns the size of the list (number of nodes)
+  size () {
+    return this.count
+  }
+
+  // Inserts a node (with the provided data) to the tail of the list
+  insert ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if tail is defined
+    if ( this.head === null ) {
+    //set oldtail next to new node
+      this.head = node
+      this.tail = node
+
+    //set newnode to tail
+    } else {
+      this.tail.next = node
+      this.tail = node
+    }
+    //set new tail to null
+    //this.tail = null
+    this.count++
+  }
+  //Returns the first node containing the provided data,
+  // or -1 if not found
+  find( value ) {
+    let currentNode = this.head
+    while (currentNode !== this.tail) {
+      if (currentNode.data === value) {
+        return currentNode
+      } else {
+        currentNode = currentNode.next
+      }
+    }
+    return currentNode.data === value ? currentNode : -1
```

Comment:
You can change the logic up to ensure that 

```javascript

while (currentNode) {
  ...
}
```

If you do it this way, you can ensure that you iterate through all the nodes in the LinkedList, and just `return -1` on line 71 instead of returning the result of your ternary -- clearer, simpler logic.